src/core/lib/experiments/experiments.yaml

```diff
@@ -206,3 +206,9 @@
   expiry: 2024/01/15
   owner: hork@google.com
   test_tags: []
+- name: rm_lookup_point_transport
+  description:
+    Change registered method's lookup point to transport
+  expiry: 2024/03/31
+  owner: yashkt@google.com
+  test_tags: []
```

Comment:
Is there a set of tests that we should run with this experiment disabled, just to make sure that path continues to work until we remove it?

Reply:
Internal would be running with the experiment disabled, so we are fine?

Reply:
We can live with just that if we have to.  But if we can identify a set of tests to run in OSS, that seems better, because then we'd detect a problem before the PR is merged instead of blocking the import.

Reply:
I can disable the experiment for  `test/cpp/end2end:end2end_test` and that gives us coverage for chttp2 and interop, so that should be good enough?

Reply:
Sounds fine.  Thanks!

Reply:
Done